doc/api/crypto.md

```diff
@@ -4187,7 +4187,7 @@ A convenient alias for [`crypto.webcrypto.getRandomValues()`][]. This
 implementation is not compliant with the Web Crypto spec, to write
 web-compatible code use [`crypto.webcrypto.getRandomValues()`][] instead.
 
-### `crypto.hash(algorithm, data[, outputEncoding])`
+### `crypto.hash(algorithm, data[, outputEncoding, outputLength])`
```

Comment:
I think for this I'd rather we support an alternative signature that accepts either an encoding for `outputEncoding` or an options object... e.g. 

```js
// current signature
crypto.hash(algorithm, data[, outputEncoding]);

crypto.hash(algorithm, data[, options]);
// where `options` is:
// { outputEncoding: '...', outputLength: ... }
```

Makes things more scalable in case additional options need to be added later.